aws-ecsfargate-terraform/README.md

```diff
@@ -1,72 +1,117 @@
-# Deploying the 1Password SCIM Bridge in AWS ECS with Terraform
+# Deploying the 1Password SCIM Bridge on AWS Fargate with Terraform
 
-This guide will run you through a deployment of the 1Password SCIM bridge to your Amazon Web Service Elastic Container Service (ECS) Fargate using Terraform. 
+This guide will run you through a deployment of the 1Password SCIM bridge on AWS Fargate using Terraform. 
 
-Note that due to the highly advanced and customizable nature of Fargate, this is only a suggested starting point. You may modify it to your needs to fit within your existing infrastructure.
+Note that due to the highly advanced and customizable nature of Amazon Web Services, this is only a suggested starting point. You may modify it to your needs to fit within your existing infrastructure.
 
 ## Prerequisites
 
 Before beginning, familiarize yourself with [PREPARATION.md](/PREPARATION.md) and complete the necessary steps there.
 
-- the [terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli) command line tools
+- [Terraform](https://www.terraform.io/downloads)
 - `scimsession` file and bearer token (as seen in `PREPARATION.md`)
-- (Optional) DNS Zone in Route53
 
-## Log In With `aws`
+## Sign in with `aws`
 
 Ensure you are authenticated with the `aws` tool in your local environment.
 
 See [Terraform AWS Authentication](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication) for more details.
 
 ## Configuration
```

Comment:
I think it may be worth mentioning any default config values when applicable in this section.

Reply:
Happy to add something here. Can you elaborate?

Reply:
I thought it may be worthwhile saying if any config values under this section have a defined default value or not, and what it is. That being said reading through the section again I realize many of the values are optional and already have a detailed description.

I'm happy with it as is.